tools/tool-base/src/main/java/io/spine/code/gen/java/TwoParagraphDoc.java

```diff
@@ -0,0 +1,45 @@
+package io.spine.code.gen.java;
+
+import com.squareup.javapoet.CodeBlock;
+import io.spine.code.javadoc.JavadocText;
+
+/**
+ * A generated Javadoc with two paragraphs.
+ */
+public abstract class TwoParagraphDoc implements GeneratedJavadoc {
```

Comment:
Seems a bit too complicated. Can we make `GeneratedJavadoc` capable of constructing a Javadoc of any length and number of paragraphs?